ui/src/components/Function/RunOutputRenderer.tsx

```diff
@@ -5,7 +5,6 @@ type RenderedData = {
   message?: string;
   errorName?: string;
   output: string;
-  status: FunctionRunStatus;
 };
 
 export default function renderRunOutput(functionRun): RenderedData {
```

Comment:
Out of scope: we should ban implicit any in this repo. Since `functionRun` is untyped, this function is polluted with `any`